app/src/main/java/pm/gnosis/heimdall/ui/dialogs/base/ConfirmationDialog.kt

```diff
@@ -0,0 +1,89 @@
+package pm.gnosis.heimdall.ui.dialogs.base
+
+import android.app.Dialog
+import android.content.Context
+import android.os.Build
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.View.SYSTEM_UI_FLAG_LAYOUT_STABLE
+import android.view.ViewGroup
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import kotlinx.android.synthetic.main.screen_confirmation_dialog.*
+import pm.gnosis.heimdall.R
+import pm.gnosis.svalinn.common.utils.getColorCompat
+
+
+class ConfirmationDialog : BaseDialog() {
+
+    interface OnDismissListener {
+        fun onConfirmationDialogDismiss()
+    }
+
+    private var dismissListener: OnDismissListener? = null
+
+    override fun onAttach(context: Context) {
+        super.onAttach(context)
+        if (context is OnDismissListener)
+            dismissListener = context
+    }
+
+    override fun onDetach() {
+        super.onDetach()
+        dismissListener = null
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setStyle(STYLE_NO_TITLE, R.style.FullScreenDialogStyle)
+    }
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        return inflater.inflate(R.layout.screen_confirmation_dialog, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        arguments?.let {
+            congratulations_text.text = getString(it[ARG_TEXT_RES] as Int)
+            congratulations_check.setImageResource(it[ARG_IMAGE_RES] as Int)
+        }
+    }
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        val dialog = super.onCreateDialog(savedInstanceState)
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            with(dialog) {
+                window?.statusBarColor = context.getColorCompat(R.color.white)
```

Comment:
Maybe put this in a utils (not necessarily in this PR) and also use it in the ViewModelActivity for `colorStatusBar`